BailOutMode/BailOutController.cs

```diff
@@ -143,11 +145,9 @@ public void OnLevelFailed()
                         return; // Don't want to repeatedly show fail effect, stop here.
 
                     //Logger.Debug("Showing fail effect");
-                    LevelFailedEffect.ShowEffect();
-                    if (Configuration.instance.FailEffectDuration > 0)
-                        StartCoroutine(hideLevelFailed());
-                    else
-                        isHiding = true; // Fail text never hides, so don't try to keep showing it
+                    GameEnergyUIPanel gameEnergyUIPanel = GameObject.FindObjectsOfType<GameEnergyUIPanel>().FirstOrDefault();
```

Comment:
This should be cached in a field on the first fail so you're not doing an expensive `FindObjectsOfType` call on every fail. (or actually, just store `failDirector` in a field if you don't use the GameEnergyUIPanel for anything else)